frontend/src/App.js

```diff
@@ -3,15 +3,17 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 import Programs from './Programs/Programs';
 import About from './Components/About';
 import Blog from './Components/Blog';
+import Home from "./Components/Home";
```

Comment:
shouldn't this suppose to be imported from './components/pages/About' './components/pages/Blog' './components/pages/Home'